package.json

```
@@ -16,13 +16,51 @@
     "type": "git",
     "url": "https://github.com/Yoast/wpseo-woocommerce"
   },
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
+    "collectCoverage": true,
+    "moduleFileExtensions": [
+      "js"
+    ],
+    "transform": {
+      "^.+\\.jsx?$": "babel-jest"
+    },
+    "transformIgnorePatterns": [
+      "[/\\\\]node_modules[/\\\\](?!lodash-es|lodash|@yoast).+\\.js$"
+    ],
+    "testRegex": "/spec/.*\\.(js)$",
```

We add the JavaScript unit tests in the `/tests/` folder in the WordPress SEO and WordPress SEO Premium plugins. For consistency with the other plugins it may be good to follow the same naming convention.
```suggestion
    "testRegex": "/tests/.*\\.(js)$",
```

---

That would make it consistent with the PHP unit tests as well. Those are also contained within a folder named `/tests/` 😉 .